locales/en/landing-all.ftl

```diff
@@ -4,6 +4,12 @@
 
 public-nav-name = { -brand-mozilla-monitor }
 
+# Top navbar
```

Comment:
```suggestion
## Top navbar

```
This applies to all strings, so it should be a group-level comment.

And add an empty group-level comment after the 4 strings to reset it.

Reply:
Updated here:
https://github.com/mozilla/blurts-server/pull/4701/commits/23831f51c683c62a6c8af7b85d437e4e31f4169a